js/load.js

```diff
@@ -65,7 +63,3 @@ window.pkp = Object.assign(PkpLoad, {
 		SubmissionsPage,
 	},
 });
-
-// Required by the URN plugin
-window.pkp.Vue.component('field-text', FieldText);
-window.pkp.Vue.component('field-pub-id', FieldPubId);
```

Comment:
This is moved to pkp/js/load.js as it needs to be inside the pkpCreateVueApp to be available in every vue instance.